src/client/components/DashboardCalendar.tsx

```diff
@@ -58,62 +60,70 @@ class CalendarEvent {
 
 function SelectableCalendar ({ localizer }: Props) {
     const [events, setEvents] = useState([{allDay:true, title: "Today"}] as CalendarEvent[]);
+    const [modalOpen, setModalOpen] = useState(false)
+
 
     // prompting user for new event
     const handleSelect = ({ start, end }) => {
-        const title = window.prompt('New Event name')
-
-        // If the title of event exists then, create the event
-        if (title) {
-            let newEvent = {} as CalendarEvent;
-            newEvent.start = moment(start).toDate();
-
-            // TODO :: Fix 'All Day' Agenda for All day events
-            newEvent.end = moment(end).add(1, "seconds").toDate();
-            newEvent.title = title;
-
-            // Tracking previous events
-            setEvents([
-              ...events,
-              newEvent
-            ])
-        }
+
+        setModalOpen(true);
+    //     const title = window.prompt('New Event name')
+    //     // If the title of event exists then, create the event
+    //     if (title) {
+
+    //         // let newEvent: CalendarEvent = {
+    //         //     start: moment(start).toDate()
+    //         //     end: moment(end).add(1, "seconds").toDate()
+    //         //     title = title
+    //         //     }
+
+    //         let newEvent = {} as CalendarEvent;
+    //         newEvent.start = moment(start).toDate();
+
+    //         // TODO :: Fix 'All Day' Agenda for All day events
+    //         newEvent.end = moment(end).add(1, "seconds").toDate();
+    //         newEvent.title = title;
+
+    //         // Tracking previous events
+    //         setEvents([
+    //           ...events,
+    //           newEvent
+    //         ])
+    //     }
```

Comment:
Let's go ahead and remove this since it will have to change the logic almost completely in order to operate with the database/graphql